src/qt/dbb_gui.cpp

```diff
@@ -970,7 +970,7 @@ void DBBDaemonGui::showIPAddressQRCode()
 
 void DBBDaemonGui::getRandomNumber()
 {
-    executeCommandWrapper("{\"random\" : \"true\" }", DBB_PROCESS_INFOLAYER_STYLE_NO_INFO, [this](const std::string& cmdOut, dbb_cmd_execution_status_t status) {
+    executeCommandWrapper("{\"random\" : \"pseudo\" }", DBB_PROCESS_INFOLAYER_STYLE_NO_INFO, [this](const std::string& cmdOut, dbb_cmd_execution_status_t status) {
```

Comment:
What's the impact of switching from `true` to `pseudo`? When will true be executed (one you connect the DBB to USB?)?